Subject: [PATCH] Uncommitted changes before Checkout at 14/07/2023, 1:10 pm [Changes]
---
Index: momentGW/tda.py
===================================================================
diff --git a/momentGW/tda.py b/momentGW/tda.py
--- a/momentGW/tda.py	
+++ b/momentGW/tda.py	
@@ -199,7 +199,8 @@
 
         p0, p1 = self.mpi_slice(self.nov)
         q0, q1 = self.mpi_slice(self.mo_energy_g.size)
-
+        print(q1-q0, self.nmo)
+        print('helo')
         # Setup dependent on diagonal SE
         if self.gw.diagonal_se:
             pq = p = q = "p"
@@ -218,6 +219,7 @@
                 Lp = self.Lpx[:, :, x]
                 eta[x, n] = lib.einsum(f"P{p},Q{q},PQ->{pq}", Lp, Lp, eta_aux) * 2.0
         cput1 = lib.logger.timer(self.gw, "rotating DD moments", *cput0)
+        #E_rpa = 0.5*(np.trace(eta[0]))
 
         # Construct the self-energy moments
         moments_occ = np.zeros((self.nmom_max + 1, self.nmo, self.nmo))
